test/functions/fcf/HigherOrderPatterns.chpl

```diff
@@ -40,7 +40,8 @@ var tests = [
 
 proc main() {
   for test in tests {
-    writeln("--- ", test:string, " ---");
```

Comment:
is there an issue somewhere that this about resolving this todo?

Reply:
Yes, I have a TODO.